.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "search.usePCRE2": true
+}
```

Comment:
Can we ignore this file?

Reply:
Yep. Was meaning to ignore that but not gotten around to it.